test/claimsTests/enrollmentClaimsTests.ts

```diff
@@ -88,7 +88,7 @@ export function enrollmentClaimsTests() {
     ) {
         const requesterDID = requesterIam.getDid();
         await requesterIam.createClaimRequest({
-            claim: { claimType, claimTypeVersion: version, fields: [] },
```

Comment:
Same question as below, but I'm not sure how this improves the tests. Does this change the tests so that they would have failed with the previous implementation?
I think what we should actually be doing is adding data to the fields and making sure that it is stripped out when we issue